code/kobe.py

```diff
@@ -1107,32 +1117,543 @@ def kobe_shadows(df_input,cdpp):
     
     return df_kobe_output, [nsystem_nonzero_views1,nsystem_nonzero_views2,nplanet_transiting]
 
+def calculate_ntransits_phase(df_kobe_output):
+    
+    """
+    Defined with KOBE version 2.0
```

Comment:
description can be more concise (pref one liner). Example:
KOBE2.0: This function calculates the number of transits seen by an observer in a Monte-Carlo manner.

NOTE should be at the end of the docstring. Also edit: In the same  KOBE shadow system.